auth/src/swig/auth.i

```diff
@@ -85,6 +85,7 @@ namespace auth {
 %ignore User::GetTokenThreadSafe;
 %ignore User::GetTokenLastResult;
 %ignore User::provider_data;
+%ignore User::provider_data_DEPRECATED;
```

Comment:
provider_data_DEPRECATED is the field for which I made a change previously. I think that User:: is the wrong namespace to ignore it.

Reply:
Uh this is weird. This section is namespaced and you are correct: it does not work here.

Changed it back